client/src/App.js

```diff
@@ -19,6 +26,24 @@ function App() {
         <main>
           <Container>
             <Switch>
+              <Route exact path="/">
```

Comment:
ok i will close this pull request and cherry pick the commit with your updates to the symptoms check to prevent the merge conflicts